src/m2n/M2N.cpp

```diff
@@ -330,6 +346,28 @@ void M2N::receive(double &itemToReceive)
   PRECICE_DEBUG("receive(double): {}", itemToReceive);
 }
 
+void M2N::receive(precice::span<double> itemsToReceive)
+{
+  PRECICE_TRACE(utils::IntraComm::getRank());
+  if (not utils::IntraComm::isSecondary()) { //coupling mode
```

Comment:
```suggestion
  if (not utils::IntraComm::isSecondary()) {
```
"coupling mode" as a term should be rather obsolete, I hope

Reply:
Yes. Copy-pasted this from somewhere else. I cleaned this up via e4c706d341bff0d5156e7918fba4d53f5afccfbf.